lib/index.js

```diff
@@ -1,13 +1,15 @@
 // Packages
 const detect = require('detect-port')
+const {coroutine} = require('bluebird')
 
 // Ours
 const serve = require('./server')
 const listening = require('./listening')
 const getModule = require('./module')
 
-module.exports = (file, flags, module = getModule(file)) => {
```

Comment:
❤️ Basically what we do in micro-dev is require the file directly and then pass a different module 😄